packages/backend/src/config/config.production.ts

```diff
@@ -38,6 +38,9 @@ export function getProductionConfig(): Config {
     syncEnabled: !getEnv.boolean('SYNC_DISABLED', false),
     freshStart: false,
     eventsSyncEnabled: false,
-    transactionCountSyncEnabled: false,
+    transactionCountSyncEnabled: getEnv.boolean(
+      'TRANSACTION_COUNT_ENABLED',
+      false,
+    ),
```

Comment:
I don't think we should use environment variables to trigger feature flags on production. It is fine for local/staging and others, but having production be as explicit as possible would be nice.